### PR TITLE
More ES6ing things

### PR DESCRIPTION
**Pull request by mmmavis**

### CODE REVIEW

File: pages/add/form/get-help-fields.js [hunk 1]
```diff
@@ -2,14 +2,16 @@ import React from 'react';
 import validator from './validator';
 import Service from '../../../js/service';
 
-let HelpTypes = React.createClass({
-  getInitialState() {
-    return {
+class HelpTypes extends React.Component {
+  constructor() {
```
Comment (Pomax) [inline pages/add/form/get-help-fields.js]: Important to use `constructor(props) { super(props); ... }` here, if you want to be able to rely on `this.props` in the component at all.

Comment (mmmavis) [inline pages/add/form/get-help-fields.js]: @Pomax I've read some articles online saying if we are not using `props` to set `state` there's no need to pass `props` to `super()`. The `App` class in `routes.jsx` seems to be functioning normally without passing `props` to `super()`.

Comment (Pomax) [inline pages/add/form/get-help-fields.js]: This is a choice. I prefer to acknowledge that the constructor is supposed to take `props`, and let the code figure out what to do based on the fact that it's getting `super(props)` called, concretely, as `super(undefined)` when we don't have a properset in the JSX that creates the object.

You can leave `props` off, but I'd prefer not to.

**Comment by mmmavis** [comment]
🍰 ☕️

**Pull request merged**